tests/test_geom_opt.py

```diff
@@ -87,6 +87,7 @@ def test_saving_traj(tmp_path):
         single_point.struct,
         write_traj=True,
         opt_kwargs={"trajectory": str(tmp_path / "NaCl.traj")},
+        traj_kwargs={"filename": str(tmp_path / "NaCl.extxyz")},
```

Comment:
Curious about why this change?

Reply:
Probably could be its own PR, but I only spotted it in testing this because it has a knock-on to things which were supposed to be in `janus_results`, unlike this.

Without this change, the default trajectory (extxyz) file was created but never cleaned up, since this test wasn't supposed to be looking at the default file location.